src/sentry/runner/commands/consumer.py

```diff
@@ -1,49 +0,0 @@
-from __future__ import absolute_import, print_function
```

Comment:
I think you need to remove this reference as well:

https://github.com/getsentry/sentry/blob/master/src/sentry/runner/__init__.py#L52

Reply:
That happened with fccd43bff8065a08faa12824fb63b70ef34423aa

Reply:
sorry, I think I was looking to one commit only.

Reply:
No worries